chains/ethereum/chain.go

```diff
@@ -125,6 +128,19 @@ func InitializeChain(chainCfg *core.ChainConfig, logger log15.Logger, sysErr cha
 		return nil, err
 	}
 
+	var forwarderClient *ForwarderClient = nil
```

Comment:
```suggestion
	var forwarderClient *ForwarderClient
```

I believe these are equivalent forms. https://play.golang.org/p/uwqE1Sz8YAR

Reply:
:+1: